### PR TITLE
LPS-144991 - Implementation of responsive menu on public navbar

### PR DESCRIPTION
**Pull request by ilzamcmed**

### CODE REVIEW

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css [hunk 1]
```diff
@@ -97,3 +97,44 @@
 			brightness(89%) contrast(83%);
 	}
 }
+
+.raylife-navbar-button{
+	border: none;
+	border: 1px solid #4C85FF !important;
+	color: #4C85FF !important;
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css:104]: Use colors from stylebook

```suggestion
        border: 1px solid var(--color-action-primary-default) !important;
        color: var(--color-action-primary-default) !important;
```

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css [hunk 2]
```diff
@@ -97,3 +97,44 @@
 			brightness(89%) contrast(83%);
 	}
 }
+
+.raylife-navbar-button{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css:101]: Spacing missing

```suggestion
.raylife-navbar-button {
```

File: modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css [hunk 3]
```diff
@@ -97,3 +97,44 @@
...
+	border: none;
+	border: 1px solid #4C85FF !important;
+	color: #4C85FF !important;
+	cursor: pointer;
+	font-size: 15px;
+	padding: 15px;
+  }
+
+.menu-drop{
```
Comment (kevenleone) [inline modules/apps/site-initializer/site-initializer-raylife/src/main/zippableResources/fragments/raylife/fragments/public-navbar/index.css:110]:
```suggestion
.menu-drop {
```

**Review by kevenleone — CHANGES_REQUESTED** [review]
@ilzamcmed can you run `gw formatSource` in site-initializer-raylife ?

Format is needed here:

Also, try `yarn format`

**Comment by kevenleone** [comment]
ci:forward

**Comment by ilzamcmed** [comment]
resent https://github.com/pt-liferay-solutions/liferay-portal/pull/696

**Pull request closed without merging**